app/state/reducers/searchFiltersReducer.js

```diff
@@ -1,12 +1,16 @@
+import moment from 'moment';
 import { handleActions } from 'redux-actions';
 
-const initialState = {
-  search: '',
-};
+function getInitialState() {
+  return {
+    date: moment().format('YYYY-MM-DD'),
```

Comment:
That's a good question. My initial thought was to always store dates as strings and only change them to moment objects where needed. Makes working with url changes and filtering easier but might make other stuff harder.
One thing to also think about is where to store the format info. The 'YYYY-MM-DD' format is for example repeated in lot of places.